connector_jira_servicedesk/models/jira_organization/adapter.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+import logging
+
+import requests
+
+from odoo.addons.component.core import Component
+
+_logger = logging.getLogger(__name__)
+
+try:
+    from jira.resources import Resource
+    from jira.utils import CaseInsensitiveDict
```

Comment:
As the class is not used, please consider to remove it.
```suggestion
```

Reply:
Branch updated